### PR TITLE
order-review-prediction model submitted

### PR DESCRIPTION
**Pull request by buraklayer**

### CODE REVIEW

**Comment by mecevit** [comment]
> I think this PR should be with the other examples on https://github.com/layerai/examples-dbt

I think someone archived that repo @maasg and moved example here.

File: common/adapter.py [hunk 1]
```diff
@@ -226,7 +226,20 @@ def _run_layer_predict(
...
+
+            # Users can use the full path for fetching models. If they are authenticated, they can also use the model
+            # name as the path. So, we try to log in and init project, only if we have the Layer api key in the dbt
+            # profile
+            if self.config.credentials.layer_api_key:
+                self.login_layer()
+                layer.init(self.get_project_name(target_node))
+
+            print(self.get_project_name(target_node))
+            print(layer_sql_function.model_name)
```
Comment (volkangurel) [inline common/adapter.py:238]: Remove these prints?

**Review by volkangurel — CHANGES_REQUESTED** [review]

**Comment by volkangurel** [comment]
@maasg yes that repo is now archived, and the examples are living within this repo

File: common/adapter.py (hunk 1 above)
Comment (mecevit) [reply common/adapter.py:238]: Done

**Review by volkangurel — APPROVED** [review]

**Pull request merged**